lib/types.ts

```diff
@@ -33,6 +33,8 @@ export interface VueTemplateCompiler {
 export interface VueTemplateCompilerOptions {
   modules?: Object[]
   outputSourceRange?: boolean
+  whitespace?: 'preserve' | 'condense'
```

Comment:
I've added it, due to compiler option type of vue-template-compiler didn't work on VSCode.

Reply:
I understand the meaning of the above comments. 😎

Reply:
This type is same as  https://github.com/vuejs/vue/blob/dev/packages/vue-template-compiler/types/index.d.ts#L6-L12

Reply:
Yeah. However, it need to be exported with vue-template-compiler.
Maybe we should export some types at the next vue 2.6.x (or 2.7).